### DISCUSSION TITLE
Java @Entity

### DISCUSSION THREAD

**Discussion by wojciechGaudnik** [mapstruct/mapstruct#3440] [Q&A] [answered]
Hi, 
I have Jpa Entity 

```java
@Entity
@AggregateRoot
@SuppressFBWarnings("EI_EXPOSE_REP2")
@SuperBuilder
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Parent extends BaseEntity {

    ...
    
    @OneToMany(mappedBy = "quiz", fetch = LAZY, cascade = ALL, orphanRemoval = true)
    private Set<Question> questions;

    ...
}
```
and
```java
@Entity
@SuppressFBWarnings("EI_EXPOSE_REP2")
@SuperBuilder
@Table(name = "qs_question")
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class Child extends BaseEntity {

    ...
    
    @Setter
    @ManyToOne(fetch = LAZY)
    @JoinColumn(name = "parent_id")
    private Parent parent;

   ...
}
```

Is there a way to use @AfterMapping to set up child back reference ? 
I'm trying with @AfterMapping but without success. I  have to catch Builder what, but when I catch builder I do not have reference to parent, maybe I'm missing something. 
Thx for help

**Comment by thunderhook** [comment, marked as answer]
You can use `@AfterMapping` on the object being built with version `1.6.0.Beta1`, see [release notes](https://github.com/mapstruct/mapstruct/releases/tag/1.6.0.Beta1) and an example inside a [unit test here](https://github.com/mapstruct/mapstruct/blob/main/processor/src/test/java/org/mapstruct/ap/test/builder/lifecycle/MappingContext.java#L82).

**Reply by wojciechGaudnik** [reply]
Done, thx.
